tests/tests/swfs/avm2/pixelbender_dithering/test.toml

```diff
@@ -4,7 +4,7 @@ num_ticks = 1
 
 [image_comparisons.output]
 tolerance = 1
```

Comment:
I wonder whether allowing a bit more tolerance and fewer outliers would make more sense. Or adding a second check, with higher tolerance, now that it's plssible... 😉 It might be the case that the nature of the test conteadicts this, dunno... But either way, it may be out of scope for this PR...

Reply:
Well... I don't think that works, I've tried it and even with tolerance=128 we have 8k outliers.

It looks like the output is just very very far off (but the visual effect looks similar).